docs/bare-metal/bare-metal.yml

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
Since this is largely the same as terraform.sample.yml, we should figure out how to make them work nicely together, rather than provide a separate file. Maybe just `-e provider=bare-metal` would be enough? Maybe with some of those other variables?